### ISSUE TITLE
Status to the network zone resource "okta_network_zone"

### ISSUE THREAD

**Issue by venkatakanupuru** [okta/terraform-provider-okta#1587]
### Community Note

- Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
- Please do not leave "+1" or other comments that do not add relevant new information or questions, they generate extra noise for issue followers and do not help prioritize the request
- If you are interested in working on this issue or have submitted a pull request, please leave a comment

### New or Affected Resource(s)

- okta_XXXXX

### Potential Terraform Configuration

```hcl
# Copy-paste your Terraform configurations here - for large Terraform configs,
# please use a service like Dropbox and share a link to the ZIP file. For
# security, you can also encrypt the files using our GPG public key.
```

### References

- #0000

Hi Team,

The resource for a Networks to add a zone,  resource "okta_network_zone" "example" {} does not have an option to include status as Active / Inactive which can be done from Okta UI, can this be included in the resource.

resource "okta_network_zone" "example" {
  name     = "example"
  type     = "IP"
  gateways = ["1.2.3.4/24", "2.3.4.5-2.3.4.15"]
  proxies  = ["2.2.3.4/24", "3.3.4.5-3.3.4.15"]
}

Steps for configuring manually :
OKTA Admin UI >> Security >> Networks >> Active or Inactive

Can you please help in including the status to this resource.

Thank you
Venkata

<img width="1015" alt="image" src="https://github.com/okta/terraform-provider-okta/assets/66885752/ebb39d63-12fe-4062-830a-c0b94260c8d1">

**Comment by duytiennguyen-okta** [comment]
OKTA internal reference https://oktainc.atlassian.net/browse/OKTA-618789

**Comment by duytiennguyen-okta** [comment]
This is fixed in the release [4.0.3](https://github.com/okta/terraform-provider-okta/releases/tag/v4.0.3) from https://github.com/okta/terraform-provider-okta/pull/1602
